### PR TITLE
react-example

### PR DESCRIPTION
**Pull request by RakshithaK06**
react example

### CODE REVIEW

File: docs/examples/react/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- **Version:** 13
+- **Build:** `npm install && npm run build`
+- **Subfolder Path:** `./docs/examples/react`
+- **Build Output Path:** `build`
+
+This will take a few minutes to deploy. Then click on the **Access** tab and copy the `External Access` URL to see the React App.
+
+## :hammer: Running locally
+
+**Pre-requisites:** Node.js v13
```
Comment (sumitkolhe) [inline docs/examples/react/README.md:23]: Node.js -> NodeJS. always use the same format

File: docs/examples/react/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- Install the dependencies
+
+```
+$ npm install
+```
+
+- To start the app
+
+```
+$ npm start
```
Comment (sumitkolhe) [inline docs/examples/react/README.md:42]: Action : static sites need a local build step too. example -

to build the app

```
$ npm build or whatever it is
```

**Review by sumitkolhe — CHANGES_REQUESTED** [review]

File: docs/examples/react/README.md (hunk 1 above)
Comment (RakshithaK06) [reply docs/examples/react/README.md:23]: Done

File: docs/examples/react/README.md (hunk 2 above)
Comment (RakshithaK06) [reply docs/examples/react/README.md:42]: re: Ok gotcha..Added.

File: docs/examples/angular/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+$ npm install
+```
+
+- Run dev server with hot reload at localhost:3000
+
+```
+$ npm start
+```
+
+- build for production with minification
```
Comment (sumitkolhe) [inline docs/examples/angular/README.md:45]: capitalize first word. build -> Build

File: docs/examples/angular/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+```
+$ npm run build
+```
+
+
+## :ambulance: Support
+
+**Contact us:** https://www.kintohub.com/contact-us
+
+**Discord:** https://discordapp.com/invite/E2CMjKP
```
Comment (sumitkolhe) [inline docs/examples/angular/README.md:56]: new discord link - > https://kintohub.com/discord

File: docs/examples/docusaurus/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+```
+$ npm run start
+```
+
+
+## :ambulance: Support
+
+**Contact us:** https://www.kintohub.com/contact-us
+
+**Discord:** https://discordapp.com/invite/E2CMjKP
```
Comment (sumitkolhe) [inline docs/examples/docusaurus/README.md:52]: new discord link - > https://kintohub.com/discord

File: docs/examples/docusaurus/README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+```
+$ npm install
+```
+
+- To start the app
+
+```
+$ npm run start
+```
```
Comment (sumitkolhe) [inline docs/examples/docusaurus/README.md:45]: Build step?

File: docs/examples/gatsby/README.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+```
+$ gatsby serve
+```
+
+
+## :ambulance: Support
+
+**Contact us:** https://www.kintohub.com/contact-us
+
+**Discord:** https://discordapp.com/invite/E2CMjKP
```
Comment (sumitkolhe) [inline docs/examples/gatsby/README.md:56]: new discord link - > https://kintohub.com/discord

File: docs/examples/ionic/README.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- Install the dependencies
+
+```
+$ npm install
+```
+
+- To start the app
+
+```
+ionic serve
```
Comment (sumitkolhe) [inline docs/examples/ionic/README.md:48]: Build step missing

File: docs/examples/ionic/README.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+## :ambulance: Support
+
+**Contact us:** https://www.kintohub.com/contact-us
+
+**Discord:** https://discordapp.com/invite/E2CMjKP
```
Comment (sumitkolhe) [inline docs/examples/ionic/README.md:55]: new discord link - > https://kintohub.com/discord

File: docs/examples/preact/README.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+```
+npm run build
+```
+
+
+## :ambulance: Support
+
+**Contact us:** https://www.kintohub.com/contact-us
+
+**Discord:** https://discordapp.com/invite/E2CMjKP
```
Comment (sumitkolhe) [inline docs/examples/preact/README.md:68]: new discord link - > https://kintohub.com/discord

File: docs/examples/react/README.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+```
+$ npm run build
+```
+
+## :ambulance: Support
+
+**Contact us:** https://www.kintohub.com/contact-us
+
+**Discord:** https://discordapp.com/invite/E2CMjKP
```
Comment (sumitkolhe) [inline docs/examples/react/README.md:55]: new discord link - > https://kintohub.com/discord

File: docs/examples/vue/README.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+
+```
+$ npm run build --report
+```
+
+## :ambulance: Support
+
+**Contact us:** https://www.kintohub.com/contact-us
+
+**Discord:** https://discordapp.com/invite/E2CMjKP
```
Comment (sumitkolhe) [inline docs/examples/vue/README.md:61]: new discord link - > https://kintohub.com/discord

File: sidebars.js [hunk 13]
```diff
@@ -106,6 +106,20 @@ module.exports = {
             'examples/laravel/laravel',
           ],
       },
+      {
+        type: 'category',
+        label: 'Static Websites',
+        collapsed: false,
+        items: [
+          'examples/react/react-example',
```
Comment (sumitkolhe) [inline sidebars.js:114]: Arrange in alphabetical order.

**Review by sumitkolhe — CHANGES_REQUESTED** [review]

File: docs/examples/angular/README.md (hunk 3 above)
Comment (RakshithaK06) [reply docs/examples/angular/README.md:45]: re: Done

File: docs/examples/angular/README.md (hunk 4 above)
Comment (RakshithaK06) [reply docs/examples/angular/README.md:56]: re: Updated

File: docs/examples/docusaurus/README.md (hunk 5 above)
Comment (RakshithaK06) [reply docs/examples/docusaurus/README.md:52]: re: Updated

File: docs/examples/gatsby/README.md (hunk 7 above)
Comment (RakshithaK06) [reply docs/examples/gatsby/README.md:56]: updated

File: docs/examples/ionic/README.md (hunk 9 above)
Comment (RakshithaK06) [reply docs/examples/ionic/README.md:55]: updated

File: docs/examples/preact/README.md (hunk 10 above)
Comment (RakshithaK06) [reply docs/examples/preact/README.md:68]: updated

File: docs/examples/react/README.md (hunk 11 above)
Comment (RakshithaK06) [reply docs/examples/react/README.md:55]: updated

File: docs/examples/vue/README.md (hunk 12 above)
Comment (RakshithaK06) [reply docs/examples/vue/README.md:61]: updated

File: sidebars.js (hunk 13 above)
Comment (RakshithaK06) [reply sidebars.js:114]: Done

File: docs/examples/docusaurus/README.md (hunk 6 above)
Comment (RakshithaK06) [reply docs/examples/docusaurus/README.md:45]: Done

File: docs/examples/ionic/README.md (hunk 8 above)
Comment (RakshithaK06) [reply docs/examples/ionic/README.md:48]: updated

**Review by sumitkolhe — APPROVED** [review]

**Pull request merged**